src/Symfony/Component/Translation/LoggingTranslator.php

```diff
@@ -83,6 +83,10 @@ public function transChoice($id, $number, array $parameters = [], $domain = null
     public function setLocale($locale)
     {
         $prev = $this->translator->getLocale();
+        if ($prev === $locale) {
+            return;
+        }
```

Comment:
I'd still call `setLocale()` on the wrapped translator in case it also has logic attached to it. But skipping the log in that situation looks like a good idea.

Reply:
Correct. Done.